packages/interactive-messages/types/lodash.isfunction.d.ts

```diff
@@ -0,0 +1,3 @@
+declare module 'lodash.isfunction' {
+  export default function(x: any): x is Function;
```

Comment:
it seems like you could have just installed `@types/lodash.isfunction`. i understand that its smaller on the bundle size to just include this here, but i feel like code duplication is a bigger burden.